### ISSUE TITLE
failure to open wms control

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#509]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/509) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

after opening and closing a couple of views and flight tracks in mscolab, I get the following error when trying to open the wms\_control for a new TopView:

```
Fatal error in MSS 1.9.1. on Linux-4.19.0-6-amd64-x86_64-with-debian-10.1
Python 3.7.0 (default, Oct  9 2018, 10:31:47) 
[GCC 7.3.0]

Please report bugs in MSS to https://bitbucket.org/wxmetvis/mss

Information about the fatal error:

Traceback (most recent call last):
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/topview.py", line 237, in openTool
    wms_cache=config_loader(dataset="wms_cache", default=mss_default.wms_cache))
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/wms_control.py", line 1587, in __init__
    parent=parent, default_WMS=default_WMS, wms_cache=wms_cache, view=view)
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/wms_control.py", line 519, in __init__
    self.wms_url_changed(self.cbWMS_URL.currentText())
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/wms_control.py", line 596, in wms_url_changed
    self.activate_wms(wms)
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/wms_control.py", line 718, in activate_wms
    self.layer_changed(0)
  File "/home/icg173/anaconda3/envs/mss-1.9.1/lib/python3.7/site-packages/mslib/msui/wms_control.py", line 870, in layer_changed
    self.parent().parent().update_predefined_maps(extra)
AttributeError: 'QListWidget' object has no attribute 'update_predefined_maps'
```

‌

**Comment by ReimarBauer** [comment]
**Original comment by Tanish Grover (Bitbucket: [tanish19](https://bitbucket.org/tanish19), GitHub: [tanish19](https://github.com/tanish19)).**

----------------------------------------

Yeah, got the same error. Will take a look at it.

‌

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

fixes error in opening topview wms control in mscolab, fixes issue #509

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in tanish19/mss/fix_issue509 (pull request #803)

fixes error in opening topview wms control in mscolab, fixes issue #509

Approved-by: Christian Rolf <c.rolf@fz-juelich.de>
